content/installation/java/package/scheduled-update-curl.md

```diff
@@ -0,0 +1,131 @@
+<!--
+title: "Schedule Java agent Updates with curl"
+description: "How to use cron and curl to schedule Java agent updates from Maven Central"
+tags: "java agent deployment maven central cron curl auto-update update"
+-->
+
+## About Updates with Maven Central 
+
+Some users like to automatically update their Contrast Java agent software to the latest version. Linux users can schedule Java agent updates from Maven Central using common Linux tools `cron` and `curl`. The following instructions show you how to configure a scheduled Java agent job on an Ubuntu 18.04 Linux host.
+
+If you want to perform each step as you follow along with this guide, you can use [Vagrant](https://www.vagrantup.com/) to create a new Ubuntu 18.04 virtual machine.
+
+```console
+vagrant init ubuntu/bionic64
+```
+
+```console
+vagrant up
+```
+
+```console
+vagrant ssh
+```
+
```

Comment:
@gilday Are there any prerequisites?

Reply:
Yes, install `vagrant`

Reply:
I put a link to https://www.vagrantup.com/

Reply:
I am going to add a prerequisite based on Tyler's feedback